#! /usr/bin/env python
# coding=UTF-8
PACKAGE = "service_demo"

from dynamic_reconfigure.parameter_generator_catkin import *
gen = ParameterGenerator()

gen.add("interval_time", int_t, 0, "interval time", 1,1,10);

exit(gen.generate(PACKAGE, "client", "set_time"))
